### PR TITLE
Created Admin login UI

### PR DESCRIPTION
**Pull request by JoelCann**
Created Admin login UI, did logic work and did API integration for that.

### CODE REVIEW

File: frontend/src/pages/AdminLogin.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  const saveLogin = async (e: React.FormEvent<HTMLFormElement>) => {
+    try {
+      e.preventDefault();
+      const formData = new FormData(e.currentTarget);
+      const email = formData.get("email") as string | null;
+      const password = formData.get("password") as string | null;
+
+      // Ensure email and password are present
+      if (!email || !password) {
+        console.error("Email and password are required.");
```
Comment (sosualfred) [inline frontend/src/pages/AdminLogin.tsx:19]: Remove console logs from the code.

File: frontend/src/pages/AdminLogin.tsx
Comment (sosualfred) [inline frontend/src/pages/AdminLogin.tsx:1]: Remove console logs from the code.

File: frontend/src/pages/login.tsx
Comment (sosualfred) [inline frontend/src/pages/login.tsx:1]: Remove console logs from the code.

File: frontend/.env.example
Comment (sosualfred) [inline frontend/.env.example:1]: Don't delete the .env.example file

**Review by sosualfred — CHANGES_REQUESTED** [review]

File: frontend/.env.example
Comment (JoelCann) [reply frontend/.env.example:1]: Yes please... I have returned it.

**Comment by sosualfred** [comment]
It seems your changes are in the frontend folder instead of the admin folder.

**Comment by JoelCann** [comment]
Yes please... I have resolved that.

**Pull request merged**